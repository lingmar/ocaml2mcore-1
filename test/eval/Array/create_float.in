let _ =
  let a = Array.create_float 5 in
  a.(0) <- 10.;
  a.(1) <- 11.;
  a.(2) <- 12.;
  a.(3) <- 13.;
  a.(4) <- 14.;
  Array.iteri (fun i x -> print_int i; print_float x; print_endline "") a
