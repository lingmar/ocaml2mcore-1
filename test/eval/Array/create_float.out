01.0e+1
11.100000e+1
21.199999e+1
31.300000e+1
41.399999e+1
